Description: Upstream changes introduced in version 3.0-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 link-monitor-applet (3.0-2) unstable; urgency=low
 .
   * debian/rules:
     - Make sure config.sub and config.guess do not end up in
       diff (Closes: #482733)
     - Switched to cdbs
     - Removed now unused LDADD and CFLAGS settings
   * Moved arch-indep files (mostly flags) to new package
     link-monitor-applet-common
   * debian/control:
     - Added Build-Depends: cdbs (>= 0.4.52)
     - Bumped Standards-Version to 3.8.0
     - Added Replaces: link-monitor-applet (<= 3.0-1) for
       link-monitor-applet-common
   * Added debian/README.source
 .
 The person named in the Author field signed this changelog entry.
Author: Adriaan Peeters <apeeters@lashout.net>
Bug-Debian: http://bugs.debian.org/482733

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

Index: link-monitor-applet-3.0/src/lm-host.gob
===================================================================
--- link-monitor-applet-3.0.orig/src/lm-host.gob	2010-04-03 17:53:03.000000000 +0200
+++ link-monitor-applet-3.0/src/lm-host.gob	2010-04-03 18:26:49.000000000 +0200
@@ -67,7 +67,7 @@
 #define DEFAULT_TIMEOUT		1000
 %}
 
-class LM:Host (abstract)
+class LM:Host from G:Object (abstract)
 {
   protected LMApplet *applet;
   property POINTER applet (link, flags = CONSTRUCT_ONLY);
@@ -143,9 +143,10 @@
     return NULL;
   }
 
-  [G_GNUC_PRINTF(3, 4)]
   private void
     set_error (self, Error error, const char *format (check null), ...)
+    attr {G_GNUC_PRINTF(3, 4)}
+
   {
     va_list args;
     char *msg;
Index: link-monitor-applet-3.0/src/lm-data-set.gob
===================================================================
--- link-monitor-applet-3.0.orig/src/lm-data-set.gob	2010-04-03 17:53:03.000000000 +0200
+++ link-monitor-applet-3.0/src/lm-data-set.gob	2010-04-03 17:53:03.000000000 +0200
@@ -48,7 +48,7 @@
 #include "lm-preferences-dialog.h"
 %}
 
-class LM:Data:Set
+class LM:Data:Set from G:Object
 {
   private LMApplet *applet;
   property POINTER applet (link, flags = CONSTRUCT_ONLY);
Index: link-monitor-applet-3.0/src/lm-shell.gob
===================================================================
--- link-monitor-applet-3.0.orig/src/lm-shell.gob	2010-04-03 18:26:40.000000000 +0200
+++ link-monitor-applet-3.0/src/lm-shell.gob	2010-04-03 18:26:49.000000000 +0200
@@ -52,7 +52,7 @@
 LMShell *lm_shell = NULL;
 %}
 
-class LM:Shell
+class LM:Shell from G:Object
 {
   public uint16_t icmp_ident;
 
Index: link-monitor-applet-3.0/src/lm-tooltips.gob
===================================================================
--- link-monitor-applet-3.0.orig/src/lm-tooltips.gob	2010-04-03 18:26:40.000000000 +0200
+++ link-monitor-applet-3.0/src/lm-tooltips.gob	2010-04-03 18:26:49.000000000 +0200
@@ -71,7 +71,7 @@
 #define GTK_PRIVATE_SET_FLAG(wid,flag)    G_STMT_START{ (GTK_PRIVATE_FLAGS (wid) |= (PRIVATE_ ## flag)); }G_STMT_END
 %}
 
-class LM:Tooltips
+class LM:Tooltips from G:Object
 {
   private GtkWidget *window;
   private TooltipsData *active_data;
